ci-operator/config/openshift/aws-load-balancer-operator/openshift-aws-load-balancer-operator-release-1.0.yaml

```diff
@@ -79,6 +79,7 @@ tests:
       OO_INSTALL_NAMESPACE: aws-load-balancer-operator
       OO_PACKAGE: aws-load-balancer-operator
       OO_TARGET_NAMESPACES: '!all'
+      OPENSHIFT_VERSION: "4.13"
```

Comment:
This seems odd that you have `OPENSHIFT_VERSION` only on aws-load-balancer-operator-release-1.0.yaml. Is it needed? Why not the other files?

Reply:
Good question, I think I ignored the other branches since they are for the TechPreview releases.

Reply:
I figured out. The TechPreview branche cannot have this because they don't have the rosa test back ported. And the `1.1` release was using 4.14 at the time the PR was created. Let me update `1.1` branch too.

Reply:
Yea that makes sense.

I suppose the point of this is to "pin" to a specific version of OpenShift in ROSA; without this, it looks like https://steps.ci.openshift.org/reference/rosa-cluster-provision installs the latest version of OCP with ROSA, which I believe we don't want.

Shouldn't we add this to `ci-operator/config/openshift/aws-load-balancer-operator/openshift-aws-load-balancer-operator-main.yaml` too?

Reply:
> I suppose the point of this is to "pin" to a specific version of OpenShift in ROSA; without this, it looks like https://steps.ci.openshift.org/reference/rosa-cluster-provision installs the latest version of OCP with ROSA, which I believe we don't want.

Right

> Shouldn't we add this to ci-operator/config/openshift/aws-load-balancer-operator/openshift-aws-load-balancer-operator-main.yaml too?

I think it's fine to keep the main branch as development one which runs on the latest OCP candidate available. This way we are sure we start new features on the latest OCP available on ROSA.

Reply:
Oh yea, that's a fair point. I think the tricky part is remembering to keep the OCP versioning up-to-date with our releases (hence the reason this PR was created).

Reply:
Yes. Similar problem we have in the operator repository with [the `latest` doc links](https://github.com/openshift/aws-load-balancer-operator/blob/main/bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml#L111).